src/main/java/org/breskul/util/AnnotationsHelper.java

```diff
@@ -0,0 +1,97 @@
+package org.breskul.util;
+
+import org.breskul.connectivity.annotation.Column;
+import org.breskul.connectivity.annotation.Id;
+import org.breskul.connectivity.annotation.Table;
+import org.breskul.exception.BoboException;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AnnotationsHelper {
+    public static String getTableName(Object object) {
+        checkIfTableAnnotationPresent(object);
+        String tableName = object.getClass().getAnnotation(Table.class).value();
+        if (tableName.isBlank()) tableName = object.getClass().getSimpleName().toLowerCase();
+        return tableName;
```

Comment:
You can make this function more laconic by using Optional:
public static <T> String getTableName(Class<T> entityType) {
return Optional.ofNullable(entityType.getAnnotation(Table.class))
.map(Table::name)
.orElseGet(() -> entityType.getSimpleName().toLowerCase());
}